data/pug/pug news.txt

```diff
@@ -0,0 +1,363 @@
+# Copyright (c) 2023 by tibetiroka
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+news "artist on post-fw Deneb"
+	location
+		government Neutral
+		system Deneb
+	name
+		word
+			"Writer"
+			"Painter"
+			"Artist"
+			"Sculptor"
+			"Designer"
+			"Photographer"
+			"Composer"
+	message
+		word
+			"This "
+		word
+			"planet"
+			"city"
+			"building"
+		word
+			"'s "
+		word
+			"alien"
+			"extraterrestrial"
+			"unique"
+			"Pug"
+		word
+			" origin is so "
+		word
+			"inspiring"
+			"intriguing"
+			"thrilling"
+			"exciting"
+		word
+			"! I'm "
+		word
+			"full of"
+			"bristling with"
+		word
+			" "
+		word
+			"ideas"
+			"creativity"
+			"energy"
+		word
+			"!"
+
+news "tourist on post-fw Deneb"
+	location
+		government Neutral
+		system Deneb
+	name
+		word
+			"Tourist"
+			"Sightseer"
+			"Traveler"
+	message
+		word
+			"Can you imagine"
+			"Did you know"
+		word
+			" that "
+		word
+			"this entire"
+			"this whole"
+			"all this"
+		word
+			" "
+		word
+			"city"
+			"building"
+		word
+			" was "
+		word
+			"constructed"
+			"created"
+			"built"
+			"made"
+		word
+			" by "
+		word
+			"an alien species"
+			"aliens"
+			"insectoid aliens"
+			"another race"
+		word
+			"? "
+		word
+			"That's"
+			"I find that"
+		word
+			" "
+			" so "
+		word
+			"inspiring"
+			"incredible"
+			"exciting"
+			"fascinating"
+		word
+			"!"
+
+news "conspiracy theorist on post-fw Deneb"
+	location
+		government Neutral
+		system Deneb
+	name
+		word
+			"Conspiracy Theorist"
```

Comment:
Should the second word be capitalized here?

Reply:
```suggestion
			"Conspiracy theorist"
```